samples/sample.js

```diff
@@ -5,39 +5,53 @@ const { Resource } = require('@opentelemetry/resources');
 const { MeterProvider } = require('@opentelemetry/sdk-metrics');
 const { configureDynatraceMetricExport } = require('..');
 
+// optional: set up logging for OpenTelemetry
 diag.setLogger(new DiagConsoleLogger(), DiagLogLevel.ALL)
 
+// Configure a MeterProvider
+const provider = new MeterProvider({
+  resource: new Resource({
+    'service.name': 'opentelemetry-metrics-sample-dynatrace'
+  })
+});
+
 const reader = configureDynatraceMetricExport(
-  // ExporterConfig
+  // exporter configuration
   {
     prefix: 'sample', // optional
-
+    defaultDimensions: [ // optional
+      { key: 'default-dim', value: 'default-dim-value' },
+    ],
     // If no OneAgent is available locally, export directly to the Dynatrace server:
     // url: 'https://myenv123.live.dynatrace.com/api/v2/metrics/ingest',
     // apiToken: '<load API token from secure location such as env or config file>'
   },
-  // ReaderConfig
+  // metric reader configuration
   {
     exportIntervalMillis: 5000
   }
 );
 
-const provider = new MeterProvider({ resource: new Resource({ 'service.name': 'opentelemetry-metrics-sample-dynatrace' }) });
 provider.addMetricReader(reader);
+
 const meter = provider.getMeter('opentelemetry-metrics-sample-dynatrace');
 
+// Your SDK should be set up correctly now. You can create instruments...
 const requestCounter = meter.createCounter('requests', {
   description: 'Example of a Counter'
```

Comment:
I think we drop the description at this point in time, so maybe we should not include it in the sample?

Reply:
Can do, we'll also need to remove it from the examples in [Java](https://github.com/dynatrace-oss/opentelemetry-metric-java/blob/fdb2a237985bb0afa73f9ec19accf61a399ff291/README.md?plain=1#L87) and [Python](https://github.com/dynatrace-oss/opentelemetry-metric-python/blob/c785fca2026736d0308e1d1fa6405d470b5ef6a7/README.md?plain=1#L43) Exporters, though.